content/blog/2022/platform-tools.md

```diff
@@ -0,0 +1,497 @@
+which provides minification
+for deployed assets like CSS.
+That seems like a great feature!
+But the specific CSS minifier that they use
+is somewhat over-eager,
+and began stripping out all my new
+[`@layer`](https://css-tricks.com/css-cascade-layers/) rules.
+I had to turn off the minification step
+in order to publish my entirely-valid CSS.
+
+That shouldn't be the case.
+Minifiers shouldn't need to understand
+all the new features that land in CSS.
+This is similar to the issue with
+[removing units from zero](/2022/08/04/zero-units/),
+or Harry Roberts' recent warnings around
+[Critical CSS](https://csswizardry.com/2022/09/critical-css-not-so-fast/).
+In a language like CSS --
+where order is meaningful,
+and duplicate properties are a useful feature --
+it's risky for automated build tools
+to try and be _clever_ about minification.
+I don't know that a CSS minifier
+should do much more than white-space management.
+
+## Other tools that might intervene
+
+In regard to web fonts,
+Bram points out
+that authors are not taking full advantage
+of variable fonts --
+we're mostly changing the font weights,
+and selecting from standard values.
+
+As someone who has spent hours
+searching through lists of variable fonts,
+that doesn't surprise me much.
+Most variable fonts are either experimental display fonts
+without a lot of practical use,
+or they only provide a single weight axis.
+While I love having all my weights in a single file
+and smoothly transitioning from one weight to another,
+_specialty weights_ are not a high priority for me.
+I'll start using more powerful font variables
+once there are more fonts available
+with more interesting variables to manipulate.
+
+Most of us are also using
+the over-complicated `font-feature-settings` property,
+instead of the simpler `font-variant`.
+I've been doing that on my site as well!
+
+Why did I think that was the best approach?
+Likely because that's the generated output
+recommended by
+[Wakamai Fondue](https://wakamaifondue.com/) --
+a popular tool for inspecting font capabilities.
+It's a great tool,
+filling a much-needed gap,
+but I wonder how the usage stats would change
+if they updated their output CSS?
+
+Maybe even more common,
+much of the web is built using CSS libraries
+like Bootstrap,
+or third-party themes
+provided by a site-builder or CMS.
+The influence of both
+WordPress and Bootstrap is clear
+throughout the Almanac data.
+When Rachel notes the popularity of
+flexbox (very popular) vs grid (not so much),
+I wonder how much of that
+can be traced back to
+Bootstrap 'grids'
+[being built in flexbox](https://getbootstrap.com/docs/5.2/layout/grid/).
+
+I don't think that's the only factor.
+It's possible authors have settled on flexbox as _good-enough_,
+and they don't think it's worth their time
+learning another technique.
+I can understand that,
+but it's unfortunate.
+I think people are missing out
+on one of the coolest features
+added to CSS _ever_.
+Oh well.
```

Comment:
```suggestion

```

I don't think you need the "Oh well" here. Don't give up on these people who haven't seen the light! They will come around.